[EN/ZH] Update `Help centre/Upgrading to lazer` & `osu! tournament client` (#14242)
ppy/osu-wiki: wiki/Help_centre/Upgrading_to_lazer/en.md

| Multiplayer commands | ![Yes][true] | ![No][false] |
| Tag co-op | ![Yes][true] | ![No][false] |
| Playlists (user-curated leaderboards) | ![No][false] | ![Yes][true] |

> | Ranked Play | ![No][false] | ![Yes][true] |

Reviewer: sentence case

```suggestion
| Ranked play | ![No][false] | ![Yes][true] |
```

| osu!mania editor | ![Yes][true] | ![Yes][true] |
| Open difficulty as reference | ![Yes][true] | ![No][false] |
| Per-object SV / volume | ![No][false] | ![Yes][true] |

> | Costom hitsound sample sets | ![Yes][true] | ![Yes][true] |

Reviewer: ```suggestion
| Custom hitsound sample sets | ![Yes][true] | ![Yes][true] |
```

However, only the following mods will award performance points for now:

*Only the **default configuration** of customisation options is eligible for performance points, unless otherwise noted.*

- Difficulty reduction
  - Easy
  - No Fail

>   - Half Time (**Only** configuring `Adjust pitch` is allowed)

Reviewer 2: Why is this "ONLY" while none of the others here have bold or even "only"?

Please do a pass on this section and tidy things up. Alternatively just explain how you can check this stuff in game rather than documenting on the wiki, since it'll probably continue to change at a rapid rate and mean this section is outdated?

Author: The `**Only**` here is meant to emphasize that the speed multiplier cannot be adjusted.
Although I personally think this `Only` is redundant given the context, considering 

> while 0.75x/1.5x is the default configuration, saying things such as "Half Time will award performance points" doesn't mention default configuration at all and rather leaves the impression that anything will work, which is not true.

it seems better to keep it.

Author: double checked and cleaned up the list. may you review it again?

- Difficulty reduction
  - Easy
  - No Fail
  - Half Time (only 0.75x, configuring `Adjust pitch` is allowed)

>   - Daycore (only 0.75x)

Reviewer 3: I think it's still necessary to either clarify the multiplier for HT/DC/DT/NC, or specify explicitly that no other adjustments than of pitch are allowed (for each of these mods, yes), or add "only configuring ... is allowed".

while 0.75x/1.5x is the default configuration, saying things such as "Half Time will award performance points" doesn't mention default configuration at all and rather leaves the impression that anything will work, which is not true.

>   - Daycore (only 0.75x)

Author: I'd prefer to move this part forward than add `only 0.75x` back, which is redundant.

> *Only the **default configuration** of customisation options is eligible for performance points, unless otherwise noted.*

- Difficulty Increase
  - Hard Rock (Not for osu!mania)
  - Sudden Death (Configuring `Also fail when missing a slider tail (only for osu!)` and `Restart on fail` is allowed)
  - Perfect (Configuring `Require perfect hits (only for osu!mania)` and `Restart on fail` is allowed)
  - Double Time (Configuring `Adjust pitch` is allowed)
  - Nightcore
  - Fade In (Only for osu!mania)
  - Hidden
  - Cover (Only for osu!mania, configuring `Direction` is allowed)
  - Flashlight
  - Blinds (Only for osu!)
  - Accuracy Challenge (All customisation options are allowed)
- Automation
  - Spun Out (Only for osu!)
- Conversion
  - Alternate (Only for osu!)
  - Single Tap (Only for osu!/osu!taiko)
  - Swap (Only for osu!taiko)
  - Mirror (Only for osu!mania)

>   - Four Keys (Only for osu!mania)

Author: applied

There is no songs folder in lazer! This allows us to do cool things like not require pressing `F5` at song select to refresh beatmaps (because beatmaps are always in a good state) and reduce the disk space used by beatmaps by 20–40%. You can read more about [the way lazer stores files](/wiki/Client/Release_stream/Lazer/File_storage).

> If you need to make changes to a beatmap, please use the in-game beatmap editor. You can alternatively click "Edit externally" under the "File" tab in the editor to open the map in a temporary folder for external editing.

Reviewer 3: ```suggestion
If you need to make changes to a beatmap, please use the in-game beatmap editor. You can alternatively click `Edit externally` under the `File` tab in the editor to open the map in a temporary folder for external editing.
```

There is no songs folder in lazer! This allows us to do cool things like not require pressing `F5` at song select to refresh beatmaps (because beatmaps are always in a good state) and reduce the disk space used by beatmaps by 20–40%. You can read more about [the way lazer stores files](/wiki/Client/Release_stream/Lazer/File_storage).

> If you need to make changes to a beatmap, please use the in-game beatmap editor. You can alternatively click `Edit externally` under the `File` tab in the editor to open the map in a temporary folder for external editing.

Reviewer 2: ```suggestion
If you need to make changes to a beatmap, please use the in-game beatmap editor. To access the raw files for a beatmap set, you can use the **Edit externally** option under the **File** menu.
```
